slapp/transforms/image_utils.py

```diff
@@ -0,0 +1,63 @@
+from typing import Tuple
+from pathlib import Path
+from PIL import Image, ImageDraw, ImageFont
+from math import ceil
+
+
+def add_scale(image_path: Path,
+              scale_position: Tuple[int, int],
+              font_file: Path,
+              ophys_experiment_resolution: float = 10/13,
+              scale_size: int = 10,
+              fill: Tuple[int, int, int] = (0, 0, 0),
```

Comment:
Nit: `fill_color` would make this clearer without having to read the docstring.

Reply:
changed to fill_color